x-pack/plugins/security_solution/public/common/containers/hosts_risk/types.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum HostRiskScoreQueryId {
```

Comment:
```suggestion
export const enum HostRiskScoreQueryId {
```

https://robinpokorny.com/blog/typescript-enums-i-want-to-actually-use/

TLDR: `enum` generates ugly runtime code. There is **no code generated** when using [`const enum`](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums). The values are inlined.